pgp/keysource_test.go

```diff
@@ -66,10 +66,27 @@ func TestGnuPGHome_Import(t *testing.T) {
 	err, _, stderr = gpgExec(gnuPGHome.String(), []string{"--list-secret-keys", mockFingerprint}, nil)
 	assert.NoErrorf(t, err, stderr.String())
 
-	assert.Error(t, gnuPGHome.Import([]byte("invalid armored data")))
+	err = gnuPGHome.Import([]byte("invalid armored data"))
+	assert.Error(t, err)
+	assert.ErrorContains(t, err, "gpg: no valid OpenPGP data found.\ngpg: Total number processed: 0\n: exit status 2")
```

Comment:
```
gpg: no valid OpenPGP data found.
gpg: Total number processed: 0
: exit status 2
```

Does still look far from optimal :grimacing:

Reply:
Yeah. I mean, we can just strip all newlines but I didn't want to change too much.